conu/apidefs/metadata.py

```diff
@@ -0,0 +1,48 @@
+class Metadata:
```

Comment:
I guess you want to inherit from `object` since we need conu to be Python 2 compatible.
See https://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object#answer-15374884